src/viam/sdk/robot/client.hpp

```diff
@@ -138,8 +185,8 @@ class RobotClient {
     void stop_all();
 
     /// @brief Cancel all operations for the robot and stop all actuators and movement.
-    /// @param extra Any extra params to pass to resources' `stop` methods, keyed by `ResourceName`.
-    void stop_all(std::unordered_map<ResourceName,
+    /// @param extra Any extra params to pass to resources' `stop` methods, keyed by `Name`.
+    void stop_all(std::unordered_map<Name,
```

Comment:
Huh. So the second parameter here is like an unwrapped `AttributeMap` that's not held by `shared_ptr`. Not asking you to change it, but I think the `AttributeMap` infra is something we should look at cleaning up.

Reply:
Oh good catch! I actually think it would be great to change this to an `AttributeMap` now while we're already changing the `RobotClient` API considerably.